### PR TITLE
Return room token

### LINKED ISSUE #4544: Return room token in occ talk:room:create
**Is your feature request related to a problem? Please describe.**
Write a shell script that creates a room with occ then adds participants.
1. Create a room with occ: `occ talk:room:create --owner admin room1`
2. Now try to add participants with `occ talk:room:add` but you don't have the token.

**Describe the solution you'd like**
`occ talk:room:create` should output the resulting room token so that the script can grab it for further use.

### PR DESCRIPTION
**Pull request by vitormattos**
close #4544

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by vitormattos** [comment]
Implementing tests and small fixes on this command on other PR

File: lib/Command/Room/Create.php [hunk 1]
```diff
@@ -156,6 +156,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			$output->writeln(sprintf('<error>%s</error>', $e->getMessage()));
```
Comment (nickvergessen) [inline lib/Command/Room/Create.php:156]: Not an error :) use info or question?

Comment (vitormattos) [reply lib/Command/Room/Create.php:156]: I don't did this. I can change but is an error, the methods addRoom* trigger errors and this row is to show the error.

Comment (nickvergessen) [reply lib/Command/Room/Create.php:156]: Nevermind wrong line

**Pull request merged**